.pre-commit-config.yaml

```diff
@@ -31,14 +31,21 @@ repos:
 - repo: local
   hooks:
     - id: watermark
-      name: Check notebooks have watermark (see Jupyter style guide from PyMC3 Wiki)
-      types: [jupyter]
+      args: [--negate, --multiline]
       entry: '%load_ext watermark.*%watermark -n -u -v -iv -w'
       language: pygrep
-      args: [--negate, --multiline]
       minimum_pre_commit_version: 2.8.0
+      name: Check notebooks have watermark (see Jupyter style guide from PyMC3 Wiki)
+      types: [jupyter]
     - id: check-toc
+      entry: python scripts/check_toc_is_complete.py
+      language: python
       name: Check all notebooks appear in table of contents
+      pass_filenames: false
       types: [jupyter]
-      entry: python scripts/check_toc_is_complete.py
+    - id: check-no-tests-are-ignored
```

Comment:
this is the only new hook here, in the others I just sorted the keys

Reply:
Good call adding this hook 👍